### ISSUE TITLE
[BUG] Get all notifications crashes with 415

### ISSUE THREAD

**Issue by aniri** [code4romania/monitorizare-vot#370]
The following endpoint is called from the web app `/api/v1/notification/get/all?page=1&pageSize=10` and it crashes with 415. 

Please note that there are no notifications currently in the DB.

**Comment by DSVDavid** [comment]
Can I take this one?

**Comment by aniri** [comment]
I didn't notice any issue with swagger before, what was the configuration that you had to change?

Please only add the fix for the endpoint in the PR and let's see if we need to make a separate PR with the swagger part :)

**Comment by DSVDavid** [comment]
> I didn't notice any issue with swagger before, what was the configuration that you had to change?
> 
> Please only add the fix for the endpoint in the PR and let's see if we need to make a separate PR with the swagger part :)

When I tried to access endpoints that require authorization on the swagger UI I usually got a 401 error. I also noticed that in the curl request printed by swagger the "Bearer" part was missing from the authorization header.

In SwaggerConfiguration.cs I changed this part:

 ```
options.AddSecurityDefinition("bearer", new OpenApiSecurityScheme
                {
                    Name = "Authorization",
                    In = ParameterLocation.Header,
                    Type = SecuritySchemeType.ApiKey
                });
                options.AddSecurityRequirement(new OpenApiSecurityRequirement
                    {
                        {
                            new OpenApiSecurityScheme
                            {
                                Reference = new OpenApiReference {Type = ReferenceType.SecurityScheme, Id = "bearer"}
                            },
                            new[] {"readAccess", "writeAccess"}

                        }
                    });
```

To this:

```
 var securitySchema= new OpenApiSecurityScheme{
                                Name="Authorization",
                                In = ParameterLocation.Header,
                                Type = SecuritySchemeType.Http,
                                Scheme = "bearer",
                                Reference = new OpenApiReference {Type = ReferenceType.SecurityScheme, Id = "Bearer"}
                };

                options.AddSecurityDefinition("Bearer",securitySchema);

                options.AddSecurityRequirement(new OpenApiSecurityRequirement
                    {
                        {
                            securitySchema,
                            new[] {"Bearer","readAccess", "writeAccess"}

                        }
                    }
                );

```

and it then worked for me.

Ok, I will do a PR for this issue and if the swagger one only happened to me for some reason I will leave it be :)

**Comment by idormenco** [comment]
Hi @DSVDavid ,
TLDR: this is not a bug this is a feature. You have to add manually `Bearer ` in front of your token when authorizing.
Long story:
1. user calls `/api/v2/access/authorize` and gets back something like
```
{
  "access_token": "<token>",
  "expires_in": 86400
}
```
2. User presses `authorize` which will add an authorization header
The problem comes in the format of bearer authorization header which is like following 
```
Authorization: Bearer <token>
````
So we manually add Bearer in front of our token .

If this is a big problem we can have a fix which you provided. But we are kinda used to do like this.
@aniri please advise.
